build/dotfile_settings.gni

```diff
@@ -6,4 +6,7 @@
 brave_exec_script_allowlist = [
   # //brave/.env parser.
   "//brave/build/env_config.gni",
+
+  # PWA manifest validation
+  "//brave/BUILD.gn",
```

Comment:
We definitely should not be adding all of //brave/BUILD.gn to the exec script whitelist, that would allow any target within it to use exec_script. Either way this seems more appropriate for presubmit